content/doc/index.html.haml

```diff
@@ -46,3 +48,50 @@ title: Jenkins Documentation
         %li
           %a{:href => url}
             = section
+
+  %div
+    %h3
+      Solution pages
+      
+    %p
+      Solution pages describe use-cases for particular Jenkins application areas.
+      
+    %ul
+      - site.solutions.keys.sort.each do |key|
+        - link = "/solutions/#{key}"
+        %li.leaf
+          %a{:href => link}
+            = site.solutions[key].usecase
+          
+  %div
+    %h3
+      Jenkins Wiki
+    
+    %p
+      The information on Wiki may be a bit outdated, but you can use it as 
+      additional information source.
+    
+    %ul
+      %li
+        %a{href: "https://wiki.jenkins-ci.org/display/JENKINS/Meet+Jenkins"}Jenkins introduction
```

Comment:
This content is already on `/download` to where I consider it the canonical source now